spec/lib/unified_health_data/service_spec.rb

```diff
@@ -127,6 +127,31 @@
       expect(service.send(:fetch_observation_value, obs)).to eq({ type: 'quantity', text: '5 mg' })
     end
 
+    it 'includes the less-than comparator in the text result when present' do
```

Comment:
maybe belt and suspenders, but i think we need a test case for `ad` [comparator](https://build.fhir.org/valueset-quantity-comparator.html).

Reply:
also, why not throw greater-than-or-equal  test in the there too :-)